Urban-Fun/Cells/TimelineCell.m

```diff
@@ -67,6 +76,25 @@ - (IBAction)didTapJoin:(id)sender {
         [Activity updateAttendanceListWithUserId:currentUser.objectId withActivity:self.activity withCompletion:^(BOOL succeeded, NSError * _Nullable error) {
             if ([self.activity.attendanceList containsObject:currentUser.objectId]){
                 [self.timelineJoinButton setSelected:YES];
+                [UIView animateWithDuration:4.0 delay:0.0 options:UIViewAnimationOptionCurveEaseOut animations:^{
+                    if ([self.activity.maxUsers intValue] > 0){
+                        if (([self.activity.attendanceList indexOfObject:currentUser.objectId] + 1 <= [self.activity.maxUsers intValue])){
```

Comment:
I didn't get to see this animation in the video you shared, would be great to include this into the test plan